fiftyone.devicedetection.onpremise/examples/onpremise/performance-console/performance.js

```diff
@@ -148,6 +148,19 @@ eventEmitter.on('FinishProcessing', (calibration) => {
     console.log(`ismobile = true : ${isMobileTrue}`);
     console.log(`ismobile = false : ${isMobileFalse}`);
     console.log(`ismobile = unknown : ${isMobileUnknown}`);
+
+    timeMsec = (actualTime - calibrationTime) / msecToNanoSec;
```

Comment:
Rather than an environment variable, I mean more the structure of the JSON and filename, as both are specific to CI.
For example, running `performance.js [datafile] [evidencefile]` outputs no JSON file. But running `performance.js [datafile] [evidencefile] [jsonfile]` outputs a more generic JSON like:
```
{
  "DetectionsPerSecond": 0.0,
  "MsPerDetection": 0.0,
  ...
}
```
which is then parsed by the script.

Even better would be named arguments e.g. `performance --datafile [datafile] --evidence [evidencefile] --jsonoutput [jsonfile]`, but that's not essential.